OJP/OJP_Trips.xsd

```diff
@@ -631,59 +631,219 @@
+			<xs:documentation>Various types of guidance advice given to traveller. Suitable values may differ by MODE (e.g. a car driver needs different advice than a person walking for a transfer.</xs:documentation>
 		</xs:annotation>
+		<!-- the classic values from OJP 1.0 with explanations -->
 		<xs:restriction base="xs:string">
-			<xs:enumeration value="origin"/>
-			<xs:enumeration value="destination"/>
-			<xs:enumeration value="continue"/>
-			<xs:enumeration value="keep"/>
-			<xs:enumeration value="turn"/>
-			<xs:enumeration value="leave"/>
-			<xs:enumeration value="enter"/>
+			<xs:enumeration value="origin">
+				<xs:annotation>
+					<xs:documentation>Defining origin.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="destination">
+				<xs:annotation>
+					<xs:documentation>Defining a destination.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="continue">
+				<xs:annotation>
+					<xs:documentation>continue on the same street/road/path.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="keep">
+				<xs:annotation>
+					<xs:documentation>keep going on the same street/road/path.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="turn">
+				<xs:annotation>
+					<xs:documentation>When this value is used, you always have to consider the value in TurnAction as well. There must be a TurnAction present if "turn" is used.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="leave">
+				<xs:annotation>
+					<xs:documentation>Can be something like an elevator or a vehicle.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="enter">
+				<xs:annotation>
+					<xs:documentation>Can be something like an elevator or a vehicle.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<!-- additional enum values for individual vehicles -->
+			<xs:enumeration value="enter_roundabout">
+				<xs:annotation>
+					<xs:documentation>Entering a roundabout.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="stay_in_roundabout">
+				<xs:annotation>
+					<xs:documentation>Staying in the roundabout.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="leave_roundabout">
+				<xs:annotation>
+					<xs:documentation>Leave the roundabout.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="enter_builtup_area">
+				<xs:annotation>
+					<xs:documentation>Entering a builtup area / community.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="leave_builtup_area">
+				<xs:annotation>
+					<xs:documentation>Leave the builtup area / community.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="ramp">
```

Comment:
Ramp is very confusing with ramp for accessibility .... I would suggest to rename it "freewayAccessRamp" (according to the  annotation)

Reply:
resoveld